Mathlib/Data/Fintype/BigOperators.lean

```diff
@@ -224,16 +224,21 @@ theorem Fintype.prod_sum_type (f : α₁ ⊕ α₂ → M) :
     ∏ x, f x = (∏ a₁, f (Sum.inl a₁)) * ∏ a₂, f (Sum.inr a₂) :=
   prod_disj_sum _ _ _
 
-@[to_additive (attr := simp) Fintype.sum_prod_type]
+/-- Product over a product type equals the product of the fiberwise products. For rewriting
+in the reverse direction, use `Fintype.prod_prod_type'`. -/
+@[to_additive Fintype.sum_prod_type "Sum over a product type equals the sum of fiberwise sums.
```

Comment:
Why has the simp attribute been removed here? If this was intentional, it should be mentioned in the PR description.

Reply:
The corresponding lemma for finsets is not tagged simp. I think either the finset lemma should be simp or neither the finset nor fintype lemma should be simp. Which do you prefer? I personally think this kind of lemmas shouldn't be simp because the RHS isn't clearly simpler than the LHS.